tensorflow/python/kernel_tests/cwise_ops_test.py

```diff
@@ -1716,14 +1734,22 @@ def _compare(self, x, use_gpu):
 
   def _testDtype(self, dtype):
     data = (np.arange(-3, 3) / 4.).reshape([1, 3, 2]).astype(dtype)
+    self._compare(data, use_gpu=False)
     self._compare(data, use_gpu=True)
-    self._compare(data, use_gpu=True)
+    #TODO: rint op is not supported for float16
+    if dtype is np.float16:
+        return
+    self._compare_rint(data, use_gpu=False)
+    self._compare_rint(data, use_gpu=True)
+    x = [0.5, 0.5000001]
+    y = [0.0, 1.0]
+    self._compare_values(x,y, use_gpu=False)
+    self._compare_values(x,y, use_gpu=True)
 
   def testTypes(self):
     for dtype in [np.float16, np.float32, np.float64]:
       self._testDtype(dtype)
 
-
```

Comment:
Don't remove the newline.  Google style guide is two blank lines between toplevel definitions in Python.